readthedocs/config/config.py

```diff
@@ -248,6 +248,21 @@ def python_full_version(self):
             )
         return ver
 
+    def as_dict(self):
+        attributes = [
```

Comment:
I wonder if this logic should live somewhere else, as a `PUBLIC_ATTRIBUTES` or something that is more likely to be updated when changed/added?